app/Console/Commands/ForceDeletePosts.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Illuminate\Console\Command;
+use Rogue\Managers\PostManager;
+
+class ForceDeletePosts extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:forceDeletePosts
+                            {type}
+                            {source}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Force deletes posts in Rogue by type and source';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(PostManager $postManager)
+    {
+        // Post Manager Instance.
+        $this->postManager = $postManager;
+
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        info('Starting to force delete posts by type and source.');
```

Comment:
Before logging, I think it'd be nice to have this ask the user for confirmation since we're wiping stuff out, something like: "Are you sure you want to delete posts of type {type} from {source}?" There are implementation details [here](https://laravel.com/docs/5.6/artisan#command-io) under "Asking For Confirmation."

Reply:
i love this, thanks for sharing @katiecrane !! Updated :)